handlers/libhoney_event_handler.go

```diff
@@ -68,6 +68,8 @@ func initLibhoney(config config.Config, version string) func() {
 	})
 
 	// configure global fields that are set on all events
+	libhoney.AddField("honeycomb.agent.name", "Honeycomb Network Agent")
+	libhoney.AddField("honeycomb.agent.version", version)
 	libhoney.AddField("honeycomb.agent_version", version)
```

Comment:
I was originally going to do that, but then I realized it was technically a breaking change if anyone is relying on the attribute in their telemetry for queries, etc. but maybe because it's namespaced and still beta it's not a big deal

Reply:
Yeah, I think it's okay to change. Let's be in charge of our own destiny :)

Reply:
[da3ddf7](https://github.com/honeycombio/honeycomb-network-agent/pull/331/commits/da3ddf794ce126c4281fc0db440d760e090520d0)